source/documentors/decisions/0002-choosing-rst.rst

```diff
@@ -0,0 +1,220 @@
+Choosing RST
+############
+
+Status
+******
+
+**Provisional**
+
+A major concern with accepting this decision is whether we can overcome the RST
+and Github barrier to entry by investing in training and proper documentation.
+This decision should be re-evaluated once we've put the relevant quickstarts,
+how-tos and references put through their paces.
+
+Context
+*******
+
+Now that we agree on the general goal of a root documentation site, we need to
+figure out how we will write the words for the site.  Per `OEP-19`_ developer
+documentation is written in RST.  However since the audience for all
+documentation is not developers it is worth re-considering and reviewing the
+alternatives and actively make a decision here rather than passively choose what
+already exists.
+
+Decision
+********
+
+We will use *Sphinx-flavored* RST for the root `docs.openedx.org`_ site and any
+documentation that is written within the repositories where the relevant
+features exist.  In particular we will use the `Sphinx`_ tool to manage our RST
+content and publish it using ReadTheDocs.org
+
+Nuance
+======
+
+This does not mean that all documentation needs to live in repositories nor
+that all of it must be RST.  There are many good reasons to write documentation
+on the wiki, in GitHub Issues, or as discussion forum posts.  Especially early
+on when there might be a lot of iteration.
+
+Concerns Considered
+===================
+
+* Git and GitHub are a barrier to entry for non-technical writers.
+   * We acknowledge that we will have to do training and write good
+     documentation to mitigate this for less technical users in our community.
+
+     Investments by GitHub as well as effort we'll be making in terms of
+     quickstarts and how-tos will help
```

Comment:
I'll note (maybe not worth mentioning though) is that the UI on the website makes it super easy to propose changes. I had no problem forking, editing, and submitting a PR entirely through the UI a few weeks ago.

Reply:
It might be worth mentioning; you really can do everything through the web UI and never actually touch command-line `git` if you choose.